First editing pass through troubleshooting guide (#17154)
metabase/metabase: docs/troubleshooting-guide/index.md

# What are you having trouble with?

- [Logging in](./loggingin.html)

- [Running Metabase](./running.html)

- [Running Metabase on Docker](./docker.html)

- [The Metabase Application Database](./application-database.html)

- [Connecting to data warehouses with Metabase](./datawarehouse.html)

- [Timezone problems](./timezones.html)

- [Can't save questions or dashboards, or getting a blank page](./proxies.html)

- [LDAP](./ldap.html)

- [Email](./email.html)

- [I think I found a bug](./bugs.html)

## Helpful tidbits

### Accessing the Metabase server logs

> You can always look for the logs that Metabase leaves on your server's file system (or wherever else you collect logs). If you are logged into Metabase with an admin account you can also access them from the Logs tab in the Troubleshooting section of the Admin Panel. To get to the Admin Panel, click the gear icon in the top-right of Metabase.

Reviewer: ```suggestion
You can always look for the logs that Metabase leaves on your server's file system (or wherever else you collect logs). If you're logged into Metabase with an admin account, you can also access the logs from the **Logs** tab in the **Troubleshooting** section of the **Admin Panel**. To get to the Admin Panel, click the **gear icon** in the top-right of Metabase.
```
